rct229/ruletest_engine/ruletest_jsons/section12/rule_12_4.json

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
Could you verify whether we have rules 12- 4, 5, 6, and 7? This is because we don't have these Python codes.

Reply:
We may need to clean up section 12 since some of the rules are changed to secondary rules. For now, we are ok with having 4, 5, 6, 7 but need to address them in the next spring.